src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
```

Comment:
its redundant

Reply:
but if i remove this part the whole style will fall

Reply:
we nead style text Roboto and 12px size